xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/main/resources/XWiki/Lightbox/WebHome.xml

```diff
@@ -0,0 +1,927 @@
+      </cache>
+      <code>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>define('lightboxTranslationKeys', {
+  prefix: 'core.viewers.attachments.',
+  keys: [
+    'date',
+    'author'
+  ]
+});
+
+require(['jquery', 'xwiki-meta', 'blueimp-gallery', 'xwiki-l10n!lightboxTranslationKeys', 'blueimp-gallery-fullscreen',
+    'blueimp-gallery-indicator'], function($, xm, gallery, l10n) {
+  var myOpenLightbox;
+  var timeout;
+
+  /*
+   * Make sure that the toolbar will remain open also while hovering it, not just the image.
+   */
+  var keepToolbarOpenOnHover = function(image) {
+    timeout = setTimeout(function() {
+      image.popover('hide');
+    }, 3000);
+    $('.popover').on('mouseenter', function() {
+      clearTimeout(timeout);
+    });
+    $('.popover').on('mouseleave', function() {
+      image.popover('hide');
+    });
+  };
+
+  /**
+   * Assign to each selected image a toolbar popover with download and lightbox options.
+   */
+  var enableToolbarPopovers = function() {
+    // Activate the lightbox for all images inside the xwikicontent.
+    $('#xwikicontent img').popover({
+      content: function() {
+        return $('#toolbarTemplate').html();
```

Comment:
``toolbarTemplate`` is too generic and thus can easily conflict with some other component that needs a toolbar template.